pages/table/stacked-and-container-variant.page.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+
+import Grid from '~components/grid';
+import Header from '~components/header';
+import Table from '~components/table';
+
+import ScreenshotArea from '../utils/screenshot-area';
+
+export default () => {
+  return (
+    <ScreenshotArea>
+      <h1>Stacked and container variants</h1>
+
+      {/* These two should look identical */}
+      <Grid gridDefinition={[{ colspan: 6 }, { colspan: 6 }]}>
+        <ExampleTable variant="container" />
+
+        <ExampleTable variant="stacked" />
+      </Grid>
+
+      <br />
+
+      <ExampleTable variant="stacked" />
+      <ExampleTable variant="stacked" />
+    </ScreenshotArea>
+  );
+};
+
+const ExampleTable = ({ variant }: { variant: 'container' | 'embedded' | 'borderless' | 'stacked' | 'full-page' }) => {
+  const [selectedItems, setSelectedItems] = useState<any>([{ name: 'Item 6' }]);
+
+  return (
+    <Table
+      variant={variant}
+      onSelectionChange={({ detail }) => setSelectedItems(detail.selectedItems)}
+      selectedItems={selectedItems}
+      ariaLabels={{
+        selectionGroupLabel: 'Items selection',
+        allItemsSelectionLabel: () => 'select all',
+        itemSelectionLabel: (selection, item) => item.name,
+      }}
+      columnDefinitions={[
+        {
+          id: 'variable',
+          header: 'Name',
+          cell: item => item.name,
+          sortingField: 'name',
+          isRowHeader: true,
+        },
+        {
+          id: 'value',
+          header: 'Value',
+          cell: item => item.value,
+          sortingField: 'value',
+        },
+      ]}
+      items={[
```

Comment:
nit: Would it be enough to use items from pages/table/generate-data and columns from pages/table/shared-configs?